docs/user/templates/webhook/01_webhook_config.md

```diff
@@ -0,0 +1,75 @@
+<!-- page-title: Webhook -->
+
+# Webhook
+
+When an event happens (e.g. creating new model's version, deploy endpoint), we might want to trigger action in other services, Merlin provides a way this by triggering a webhook call. 
+The webhook that will be called is configured per event type, and it could contain multiple endpoints which can be configured as synchronous or asynchronous call. 
+
+For the detail of available configuration please refer to [MLP library docs](https://github.com/caraml-dev/mlp/blob/main/api/pkg/webhooks/README.md), this documentation will only cover the basic config and the paylod of webhook which we has implemented.
+
+## 1. Webhook Configuration
+In Merlin's yaml for config you can add new field for the webhook's configuration which will looks like this:
+```yaml
+WebhooksConfig:
+  Enabled: true
+  Config:
```

Comment:
Can we give an example of `on-model-version-predeployment` here too? Just so it's clearer that this webhook exists.